README.md

```diff
@@ -61,6 +61,7 @@ Options
 
 Release History
 ---------------
+- 2015-11-05    `0.5.2` - The reporter will create the output filePath parent folder, if it does not exists yet;
```

Comment:
Please remove this line, the release will be created after the code applies to the coding standard